config/manager/kustomization.yaml

```diff
@@ -6,11 +6,11 @@ generatorOptions:
 
 configMapGenerator:
 - files:
-  - keda_operator_config.yaml
+  - keda_manager_config.yaml
   name: manager-config
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
   newName: k3d-kyma-registry:5001/keda-manager-dev-local
-  newTag: 0.0.2
+  newTag: 0.0.1
```

Comment:
I see inconsistent version numeration between this file and `hack/common/Makefile`.